python/cugraph/cugraph/tests/utils/test_dataset.py

```diff
@@ -94,7 +95,10 @@ def setup_deprecation_warning_tests():
 # Helpers
 
 # check if there is a row where src == dst
-def has_loop(df):
+def has_selfloop(dataset):
+    if not dataset.metadata["is_directed"]:
+        return False
+    df = dataset.get_edgelist(download=True)
```

Comment:
Some of the newer datasets added here a quite large. Can we still get decent coverage of the code added without testing every one?

Reply:
Yes. In order to avoid downloading the large datasets along with our old ones every time for testing, the benchmark datasets are all tested separately in a single unit test.